bin/update_web_fonts_metrics.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Includes script-based language letters metrics (only implemented for Arabic) into fontconfig.json
+# to make them available after compilation in MetaApps when working in browser (JS target).
+
+SCRIPT_DIR=$( cd "$( dirname "$0" )" && pwd -P )
+BASE_DIR=$( cd "$( dirname "$SCRIPT_DIR" )" && pwd -P )
+
+pushd $BASE_DIR/resources/webfonts/src
+npm install
```

Comment:
Is it worth it to do `nmp install` here? Maybe it's better to put that in the readme as some prereq for running this script? Including node version.

Reply:
Added this of failproof thoughts. If already installed, it takes a couple of seconds, if not, installs.